gtk/details.cc

```diff
@@ -12,6 +12,8 @@
 #include <stddef.h>
 #include <stdio.h> /* sscanf() */
 #include <stdlib.h> /* abort() */
+#include <string_view>
+#include <typeinfo>
```

Comment:
Why do we need to include `<typeinfo>` everywhere? I don't see it being used...

Reply:
It's for [std::literals::string_view_literals::operator""sv](https://en.cppreference.com/w/cpp/string/basic_string_view/operator%22%22sv).